pkg/instrumentation/defaultinstrumentation_test.go

```diff
@@ -39,27 +39,29 @@ func Test_getDefaultInstrumentation(t *testing.T) {
 			Java: v1alpha1.Java{
 				Image: defaultJavaInstrumentationImage,
 				Env: []corev1.EnvVar{
-					{Name: "OTEL_SMP_ENABLED", Value: "true"}, //TODO: remove in favor of new name once safe
-					{Name: "OTEL_AWS_APP_SIGNALS_ENABLED", Value: "true"},
+					{Name: "OTEL_AWS_APP_SIGNALS_ENABLED", Value: "true"}, //TODO: remove in favor of new name once safe
```

Comment:
Do we still need this comment?

Reply:
Yes, all `AWS_APP_SIGNALS` will be replaced by `AWS_APPLICATION_SIGNALS`